src/plugins/telemetry/common/constants.ts

```diff
@@ -49,7 +49,7 @@ export const LOCALSTORAGE_KEY = 'telemetry.data';
 /**
  * Link to Advanced Settings.
  */
-export const PATH_TO_ADVANCED_SETTINGS = 'management/kibana/settings';
+export const PATH_TO_ADVANCED_SETTINGS = '/app/management/kibana/settings';
```

Comment:
nit -> I think is better to remove the first trailing slash (`app/management/kibana/settings`) and leave it to the consumer to add it. It is better than having to remove it.